pkg/tests/cli_test.go

```diff
@@ -82,17 +82,17 @@ func TestVerify(t *testing.T) {
 	tsrPath := getTimestamp(t, restapiURL, artifactContent)
 
 	// write the cert chain to a PEM file
-	pemPath := getCertChainPEM(t, restapiURL)
+	pemPath := writeCertChainToPEMFile(t, restapiURL)
 
 	// It should verify timestamp successfully.
-	out := runCli(t, "--timestamp_server", restapiURL, "verify", "--timestamp", tsrPath, "--artifact", artifactPath, "--cert-chain", pemPath)
+	out := runCli(t, "--timestamp_server", restapiURL, "verify", "--timestamp", tsrPath, "--artifact", artifactPath, "--certificate-chain", pemPath)
```

Comment:
Can we test all flags, nonce, OID, common-name, etc? And also a test with --certificate?